pkg/cloudprovider/controller/nodecontroller.go

```diff
@@ -104,6 +107,14 @@ func NewNodeController(
 	nodeMonitorGracePeriod time.Duration,
 	nodeStartupGracePeriod time.Duration,
 	nodeMonitorPeriod time.Duration) *NodeController {
+	eventBroadcaster := record.NewBroadcaster()
+	recorder := eventBroadcaster.NewRecorder(api.EventSource{Component: "controllermanager"})
+	if kubeClient != nil {
+		glog.Infof("Sending events to api server.")
+		eventBroadcaster.StartRecordingToSink(kubeClient.Events(""))
```

Comment:
If we're going to do that (which I don't necessarily object to), we should move everything to use cluster-id at the same time. All of our other event generation stuff currently seems to be using empty namespace, so for this PR it's probably best to leave as-is.

Examples:
https://github.com/GoogleCloudPlatform/kubernetes/blob/894fbc1e8a0522a63ea4880bbc29f1439de638e8/plugin/cmd/kube-scheduler/app/server.go#L100
https://github.com/GoogleCloudPlatform/kubernetes/blob/8599564c96083aa221ebce070b67b22d5bbd9b1e/cmd/kubelet/app/server.go#L404